connector_salesforce/__openerp__.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Author: Nicolas Bessi
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+{'name': 'Salesforce Connector',
+ 'version': '0.1',
+ 'author': 'Camptocamp,Odoo Community Association (OCA)',
+ 'maintainer': 'Camptocamp,Odoo Community Association (OCA)',
+ 'category': 'Connector',
```

Comment:
I dont like the category connector in general (too generic)
Customer Relationship Management or CRM?